.circleci/config.yml

```diff
@@ -38,16 +38,20 @@ parameters:
   GITHUB_ACCESS_TOKEN:
     type: string
     default: ""
+  NODE_VERSION:
+    type: string
+    default: "22.4.1"
 
 jobs:
   publish:
     docker:
-      - image: cimg/node:22.4.1
+      - image: cimg/node:<< pipeline.parameters.NODE_VERSION >>
```

Comment:
since the execution side is limited to circleci not worried about having an env control image but you think it would be more consistent to have a more comprehensive base image with nvm?